packages/block-editor/src/components/inserter/block-list.js

```diff
@@ -201,9 +200,11 @@ function InserterBlockList( {
 	}, [ filterValue, items, categories, collections ] );
 
 	const childItems = useMemo( () => {
-		return filter( filteredItems, ( { name } ) =>
-			includes( rootChildBlocks, name )
-		);
+		return rootChildBlocks
+			? filter( filteredItems, ( { name } ) =>
+					rootChildBlocks.includes( name )
+			  )
+			: [];
```

Comment:
I added in an optimization here so that if `rootChildBlocks` is nullish, the filter will not run and an empty array will be returned. Before, it would have run the filter callback multiple times even though it would always return `false`.